app/views/stages/_fields.html.erb

```diff
@@ -1,92 +1,34 @@
 <fieldset>
-  <div class="form-group">
-    <%= form.label :name, "Name", class: "col-lg-2 control-label" %>
-    <div class="col-lg-4">
-      <%= form.text_field :name, class: "form-control" %>
-    </div>
-  </div>
-
-  <% if form.object.persisted? %>
-    <div class="form-group">
-      <%= form.label :permalink, "Permalink", class: "col-lg-2 control-label" %>
-      <div class="col-lg-4">
-        <%= form.text_field :permalink, class: "form-control" %>
-      </div>
-    </div>
-  <% end %>
-
-  <div class="form-group">
-    <%= form.label :notify_email_address, "Email addresses", class: "col-lg-2 control-label" %>
-    <div class="col-lg-4">
-      <%= form.text_field :notify_email_address, class: "form-control" %>
-      <span class="help-block">Enter the email addresses that should be notified of new deploys, separated by <code>;</code></span>
-    </div>
-  </div>
+  <%= form.input :name %>
+  <%= form.input :permalink if form.object.persisted? %>
+  <%= form.input :notify_email_address,
+        label: "Email addresses",
+        help: "Enter the email addresses that should be notified of new deploys, separated by <code>;</code>".html_safe
+  %>
 
   <% if DeployGroup.enabled? %>
     <%= render 'deploy_groups/deploy_group_select', form: form %>
   <% else %>
-    <div class="form-group">
-      <div class="col-sm-offset-2 col-sm-10">
-        <div class="checkbox">
-          <%= form.label :production do %>
-            <%= form.check_box :production %>
-            Does this stage impact production servers?
-          <% end %>
-        </div>
-      </div>
-    </div>
+    <%= form.input :production, as: :check_box,
+          label: 'Impacts production',
+          help: "Deploys to production need to pass a buddy check when BuddyCheck is enabled."
```

Comment:
... I still want to allow people to set this even if they don't have buddy checks ... as documentation / for later use ... leaving it a bit ambiguous so I don't have to add extra logic :)